workspaces/ui-v2/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM library/nginx:alpine
-ARG APP
-COPY build-${APP}/ /usr/share/nginx/html/
```

Comment:
based on the inconsistent naming of our repos, I thought it was better to remove that assumption